tests/wpt/mozilla/meta/MANIFEST.json

```diff
@@ -13052,6 +13052,12 @@
      {}
     ]
    ],
+   "mozilla/fetch_cannot_overwhelm_system.window.js": [
+    [
+     "/_mozilla/mozilla/fetch_cannot_overwhelm_system.window.html",
```

Comment:
Could you run `./Mach update-manifest`?

Reply:
I'm looking into it, it seems to not have any effect...

Reply:
Try manually removing the `/_mozilla/` prefix, in that case.

Reply:
Ok, I've filed https://github.com/servo/servo/issues/23214